File: asalSayıBulma.c
#include <stdio.h>

int main()
{
  int sayı;
  int durum = 1;

 printf("Sayı giriniz :");              scanf("%d",&sayı);

 for(int i=2;i<sayı;++i)
 {
  if(sayı%i==0)
  {
     durum = 1;
     break;
 }
 else{
  durum = 0;
}
}
if(durum == 0) printf("Girdiginiz sayı asaldır");
if(durum == 1) printf("Girdiniz sayı asal degildir");

 return 0;
}
 

File: asalSayıBulma.py
durum = 1

sayı1 = int(input("Bir sayı giriniz :"))

for i in range(2,sayı1):
    
    if sayı1%i == 0:
        durum = 1
        break
    else:
        durum = 0
        
if durum == 1:
    print("Asal Sayı Değildir")
else:
    print("Asal Sayıdır")
